Tools-Override/CloudTest.targets

```diff
@@ -493,10 +476,9 @@
         <Build Condition="'$(IsOfficial)'=='true'">$(OfficialBuildId)</Build>
         <Type>$(HelixJobType)</Type>
         <Source>$(HelixSource)</Source>
-        <Properties>{ &quot;architecture&quot; : &quot;$(Platform)&quot;, &quot;configuration&quot;: &quot;$(ConfigurationGroup)&quot;, &quot;operatingSystem&quot; : &quot;$(TargetQueue)&quot; }</Properties>
+        <Properties>{ &quot;architecture&quot; : &quot;$(ArchGroup)&quot;, &quot;configuration&quot;: &quot;$(ConfigurationGroup)&quot;, &quot;operatingSystem&quot; : &quot;$(TargetQueue)&quot; }</Properties>
```

Comment:
Yup.  This file is full of said CoreFX-isms.  I intend to redo this soon;  these properties should come from whoever invokes the Cloud Test target code, we should simply check that values for required properties are provided and nothing else.